src/options.rs

```diff
@@ -61,9 +61,20 @@ pub struct Args {
     #[clap(long, default_value_t = 9.0, help_heading = "IMAGE")]
     pub aspect_height: f64,
 
-    /// The theme to use for rendering. Use `foo` to see a list of possible values
-    #[clap(long, default_value = "Solarized (dark)", help_heading = "COLORS")]
-    pub theme: String,
+    /// The themes to use for rendering. Use `foo` to see a list of possible values, defaulting to "Solarized (dark)"
+    ///
+    /// If multiple are specified, the output file name will be adjusted to match the theme accordingly.
+    #[clap(long, help_heading = "COLORS")]
+    pub theme: Vec<String>,
```

Comment:
Is it not possible for clap to include a default value that is a `Vec` with one value in it?

Reply:
That's a great catch! Initially I couldn't get it to work, but now I checked again and it indeed is quite possible! Much better, thanks!